templates/webapi/test/infopanel.html.ep

```diff
@@ -79,20 +80,20 @@
                             <span>Skip children</span>
                             %= end
                             %= link_post url_for('apiv1_restart', jobid => $testid)->query({skip_ok_result_children => 1}) => (class => 'dropdown-item restart-result', 'data-remote' => 'true', id => 'restart-result-skip-ok-children', 'data-jobid' => $testid) => begin
-                            <span>Skip restarting "OK" children</span>
+                            <span>Skip restarting "OK" (<%= $ok_results_explanation %>) children</span>
```

Comment:
Or just
```suggestion
                            <span>Skip restarting <%= $ok_results_explanation %> children</span>
```
which would render to "Skip restarting passed/softfailed children" so we wouldn't bother the users with our internal abstraction/generalization at all?

Reply:
@kalikiana  Thanks for involving me and delivering the PR. I am not quite familiar with code in this directory, but overall it looks good :-)

Reply:
This really just documentation/ UX. The important thing is that it makes sense to users - I guess it may not be easy to get from the template files